start_gomoku.sh

```diff
@@ -36,29 +38,37 @@ PYTHON=src/PyInterface/gomoku.py
 echo "Target dylib is expected to by found at the following path: $DYLIB"
 
 # Build
-maturin develop --release
-
 # Check if build is successful
-if [ $? -ne 0 ]; then
-    echo -e "\033[0;31mMaturin failed to build the project! Will NOT run the project\033[0m"
+if ! maturin develop --release; then
+    echo -e "\033[0;31mMaturin failed to build the project in a venv.\033[0m Will try to build without a virtual environment"
+    else
+      HAS_VENV=true
+    if ! pip3 install -r requirements.txt; then
+    echo -e "\033[0;31mCould not install requirements. Cannot build project\033[0m"
     exit 42
+    else
+      HAS_VENV=false
+    fi
```

Comment:
Maybe an indent issue here (the first `else` should not be there)

Reply:
Fixed

Reply:
That's a really nice fix 😂